src/formatter/Cli.hx

```diff
@@ -157,14 +157,17 @@ class Cli {
 			#end
 
 			if (content == null) {
+				Sys.stderr().writeString("Could not read anything from STDIN");
 				Sys.exit(-1);
 			}
 			if (paths.length != 1) {
 				Sys.println(content);
+				Sys.stderr().writeString("Please use exactly one `--source <path>` parameter");
```

Comment:
Should perhaps say "with --stdin" so it's clear that's only a requirement there?